languages/csharp/exercises/concept/lists/.docs/introduction.md

```diff
@@ -0,0 +1,7 @@
+Lists in C# are collections of primitive values or instances of structs or classes. They are implemented in the base class library as `List<T>` where `T` is the type of the item in the list. The API exposes a rich set of methods for creating and manipulating lists.
+
+```csharp
+var list = new List<string>();
+```
+
+A collection definition typically includes a place holder in angle brackets, often `T` by convention. This allows the collection user to specify what type of items to store in the collection. In the above example code we are instantiating a list of strings.
```

Comment:
I think maybe it could be useful to mention that `List<T>`is referred to as a generic type and that `T` is the type parameter. This is the way the [official docs](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/generics/generic-type-parameters) describe generic types.

Reply:
Done